calyx-ir/src/control.rs

```diff
@@ -37,6 +56,25 @@ impl GetAttributes for Par {
     }
 }
 
+// Data for the `par` control statement.
+#[derive(Debug)]
+pub struct StaticPar {
+    /// List of `StaticControl` statements to run in parallel.
+    pub stmts: Vec<StaticControl>,
+    /// Attributes attached to this control statement.
+    pub attributes: Attributes,
+    /// Latency, in cycles
+    pub latency: u64,
+}
+impl GetAttributes for StaticPar {
+    fn get_attributes(&self) -> &Attributes {
+        &self.attributes
+    }
+    fn get_mut_attributes(&mut self) -> &mut Attributes {
+        &mut self.attributes
+    }
+}
```

Comment:
misc note: feels like we should just make a procedural macro for this since this exact snippet gets repeated in a bunch of places